packages/cfd/src/Components/cfd-download-container.tsx

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import classnames from 'classnames';
+import { QRCode } from 'react-qrcode';
+import { Icon, Text, DesktopWrapper } from '@deriv/components';
+import { CFD_PLATFORMS, isDesktop, isMobile } from '@deriv/shared';
+import { Localize, localize } from '@deriv/translations';
+import {
+    getDXTradeWebTerminalLink,
+    getPlatformDXTradeDownloadLink,
+    getPlatformMt5DownloadLink,
+} from '../Helpers/constants';
+import { general_messages } from '../Constants/cfd-shared-strings';
+import { TCFDDashboardContainer } from './props.types';
+
+const DxtradeDesktopDownload = ({ active_index, dxtrade_tokens }: Partial<TCFDDashboardContainer>) => {
+    return (
+        <>
```

Comment:
```suggestion
        <React.Fragment>
```
This is part of our codebase consistency. Please update the other fragments to be in this format as well

Reply:
Sorted.